### PR TITLE
feat(chart): add extraVolumes / extraVolumeMounts to support a custom CA

### LINKED ISSUE #136: Allow passing a custom CA via volumes on the Helm Chart.
### Description
Hi @sergelogvinov, i'm testing the PMX CCM + CSI combo on my cluster which uses a custom CA for the web endpoint.
By the moment only the CCM Helm chart supports passing the CA via a secret, so it's just matter of porting the changes to the CSI one.

I have successfully tested this adding the CA to the controller and node containers in the deployment templates.

### PR DESCRIPTION
**Pull request by mlinares1998**
# Pull Request

## What? (description)
Add extraVolumes / extraVolumeMounts to support a custom CA on the Helm chart.

It's a port of the changes merged by @tamcore in the Proxmox CCM repo.
https://github.com/sergelogvinov/proxmox-cloud-controller-manager/pull/67

## Why? (reasoning)
Allows mounting a custom CA certificate to `/etc/ssl/certs/ca-certificates.crt`.
Resolves #136 
## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you linted your code (`make lint`)
- [ ] you linted your code (`make unit`)

> See `make help` for a description of the available targets.

### CODE REVIEW

**Comment by sergelogvinov** [comment]
Hello, thank you!

can you change commit message, and run `make docs` to regenerate yaml-files

Thanks.

File: charts/proxmox-csi-plugin/templates/node-deployment.yaml [hunk 1]
```diff
@@ -125,6 +128,9 @@ spec:
           hostPath:
             path: /sys
             type: Directory
+        {{- with .Values.extraVolumes }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
```
Comment (sergelogvinov) [inline charts/proxmox-csi-plugin/templates/node-deployment.yaml:133]: Node daemonset does not call proxmox directly.
I think the extra volumes does not need here...

Thanks.

**Comment by sergelogvinov** [comment]
Can you squash the commits in one?

File: charts/proxmox-csi-plugin/templates/node-deployment.yaml (hunk 1 above)
Comment (mlinares1998) [reply charts/proxmox-csi-plugin/templates/node-deployment.yaml]: Tested it and everything seems to work fine.
I squashed all the commits into a single one with all the changes.

Thanks!

**Comment by sergelogvinov** [comment]
Thank you!

**Pull request merged**